front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -333,7 +334,7 @@ const importTimetable = async (
         }
         const DTOLabel = DTOLabels.find((DTOlabel) => DTOlabel.label === label);
         if (DTOLabel) {
-          return DTOLabel;
+          formatedLabels.push(DTOLabel);
         }
```

Comment:
Isn't there an `else` missing here? Before, there was a return, so the iteration ended in the if.

Reply:
Done